plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -20,12 +39,18 @@
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineStandardWizardPage page;
+  private WizardPage page;
```

Comment:
So maybe we should hold onto both the `AppEngineStandardWizardPage` and the `AppEngineComponentPage`.  That would avoid the `instanceof` checking elsewhere.